client/components/SampleCensus.js

```diff
@@ -528,7 +528,7 @@ const sampleCensusButtons = [
     button: 'components.SampleCensus.preliminary_button',
     description: 'components.SampleCensus.preliminary_description',
     questions: PRELIMINARY_QUESTIONS,
-    icon: '',
+    icon: ''
```

Comment:
just a note, I prefer to leave the trailing commas in so that if new change comes here it does look as confusing but this is only for git pr reviewing.